docs/_docs/05-configuration.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "BeSLab Configuration File"
+permalink: /docs/configuration/
+excerpt: "Settings for configuring and customizing the theme."
+last_modified_at: 2021-05-11T10:40:42-04:00
+toc: true
+---
+
+## Description
+
+This configuration file, named `genesis.yaml`, is utilized by the BLIman (Be-Secure Lab Instantiator Manager) tool to seed a BeSLab instance. BeSLab is a platform designed for setting up security analysis labs efficiently.
+
+## Usage
+
+The `genesis.yaml` file contains various configuration options that define the setup of the BeSLab instance. These options include specifying the lab type, lab name, version of BeSMan (Be-Secure Management), deployment mode, version of BeSLab, URLs and paths for retrieving necessary files, configuration for Java, dashboard tool settings, and more.

```

Comment:
Add Code Collaboration Tool as well. Code Collaboration tool is the main component of BeSLab.